images/logging-eventrouter.yml

```diff
@@ -1,15 +1,28 @@
+content:
+  source:
+    dockerfile: Dockerfile
```

Comment:
Note that this process ends up not only modifying the Dockerfile downstream, but also will modify (via PR) the upstream Dockerfile with the corresponding CI pullspecs for the builder and base.  If you want to leave your existing Dockerfile as it is for the community, then you can make a separate Dockerfile.rhel with the appropriate `FROM`s and use it here.

Reply:
thx